<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Picker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>371</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="rgb_block">
         <property name="title">
          <string>RGB</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QSpinBox" name="r_rgb_spin">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>B:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSlider" name="r_rgb_slider">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>G:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="g_rgb_spin">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="g_rgb_slider">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="b_rgb_spin">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSlider" name="b_rgb_slider">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>R:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="lab_block">
         <property name="title">
          <string>LAB</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>L:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="l_lab_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSlider" name="l_lab_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>a:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="a_lab_spin">
              <property name="minimum">
               <number>-128</number>
              </property>
              <property name="maximum">
               <number>128</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="a_lab_slider">
              <property name="minimum">
               <number>-128</number>
              </property>
              <property name="maximum">
               <number>128</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>b:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="b_lab_spin">
              <property name="minimum">
               <number>-128</number>
              </property>
              <property name="maximum">
               <number>128</number>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSlider" name="b_lab_slider">
              <property name="minimum">
               <number>-128</number>
              </property>
              <property name="maximum">
               <number>128</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="cmyk_block">
         <property name="title">
          <string>CMYK</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="3" column="1">
             <widget class="QSpinBox" name="k_cmyk_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSlider" name="y_cmyk_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="m_cmyk_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="c_cmyk_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="y_cmyk_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="m_cmyk_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>K:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>M:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QSlider" name="k_cmyk_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSlider" name="c_cmyk_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>C:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="hsv_block">
         <property name="title">
          <string>HSV</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>H:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="h_hsv_spin">
              <property name="maximum">
               <number>360</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSlider" name="h_hsv_slider">
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>S:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="s_hsv_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="s_hsv_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>V:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="v_hsv_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSlider" name="v_hsv_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="xyz_block">
         <property name="title">
          <string>XYZ</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="2" column="1">
             <widget class="QSpinBox" name="z_xyz_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="x_xyz_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="y_xyz_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSlider" name="x_xyz_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSlider" name="z_xyz_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="y_xyz_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="hsl_block">
         <property name="title">
          <string>HSL</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>H:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="h_hsl_spin">
              <property name="maximum">
               <number>360</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSlider" name="h_hsl_slider">
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>S:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="s_hsl_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="s_hsl_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>L:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="l_hsl_spin">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSlider" name="l_hsl_slider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="selectedColor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Palette" name="colorPicker">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="selectedColorHex">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 2px solid gray;</string>
    </property>
    <property name="text">
     <string>#000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>311</width>
      <height>98</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Предупреждение:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Palette</class>
   <extends>QLabel</extends>
   <header>palette.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
